config/locales/en.yml

```diff
@@ -1021,6 +1021,7 @@ en:
         case_type_fields:
           case_type: Case type
           case_type_hint: For example Trial
+          select_option: Please select
```

Comment:
**not a blocker** content wise this might need to not use please. but not sure - i was told ages ago not to use please (hints generally say Choose an option or, more specifically "choose a case type"

Reply:
> **not a blocker** content wise this might need to not use please. but not sure - i was told ages ago not to use please (hints generally say Choose an option or, more specifically "choose a case type"

Cheers, currently being reviewed by Alan and Dan.